x-pack/plugins/infra/public/routes.tsx

```diff
@@ -20,15 +21,17 @@ interface RouterProps {
 
 export const PageRouter: React.SFC<RouterProps> = ({ history }) => {
   return (
-    <Router history={history}>
-      <Switch>
-        <Redirect from="/" exact={true} to="/home" />
-        <Route path="/logs" component={LogsPage} />
-        <Route path="/home" component={HomePage} />
-        <Route path="/link-to" component={LinkToPage} />
-        <Route path="/metrics/:type/:node" component={MetricDetail} />
-        <Route component={NotFoundPage} />
-      </Switch>
-    </Router>
+    <I18nProvider>
```

Comment:
It's wrong place to wrap with `I18nProvider`. Please find the root React element.